packages/frontend/app/components/curriculum-inventory/report-header.js

```diff
@@ -6,11 +6,10 @@ import { restartableTask } from 'ember-concurrency';
 
 @validatable
 export default class CurriculumInventoryReportHeaderComponent extends Component {
-  @NotBlank() @Length(3, 200) @tracked name;
+  @NotBlank() @Length(3, 200) @tracked name = this.getName();
```

Comment:
I've been using [constructor setting](https://github.com/ilios/frontend/blob/8afe1112f474375f435a3cc9cb0a833397aaa86b/packages/frontend/app/components/school-manager.js#L14C1-L17C4) for these - 

```
  constructor() {
    super(...arguments);
    this.name = this.args.report.name;
  }
```

I'm not sure if one is better better, up for discussion.

Reply:
Yes, this is better. Will change.

Reply:
I believe it's better because it's using a standard class creation pattern, rather than a specific custom function.